Documentation/cockroachdb.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: CockroachDB
-weight: 27
+weight: 5
```

Comment:
were you able to run all these changes locally to test the navigation/layout?

Reply:
yep, thanks to @galexrt's instructions